### PR TITLE
[WIP] Data Store Charter

### PR DESCRIPTION
**Pull request by kozbo**
Charter for the Data Store group. 

Todo:
 - [ ] flesh out each section in the "in-scope" section
 - [ ] make sure that all items are actual deliverables and not day-to-day operational items. On-going tasks cannot be completed as part of the charter. They belong as part of the operations of running a Data Store instance.

### CODE REVIEW

File: charters/DataStore/DataStore.md [hunk 1]
```diff
@@ -0,0 +1,58 @@
+
```
Comment (brianraymor) [inline charters/DataStore/DataStore.md:1]: The path should be charters/DataStore/charter.md.

File: charters/DataStore/DataStore.md [hunk 2]
```diff
@@ -0,0 +1,58 @@
...
+* EOY 2018: add checkout, collections, improved scaling/hardening, generic events to support stand-alone indexers, additional gaps identified in HCA DCP Pilot.
+* Future: native GCP support, AuthZ support for controlled-access data, additional scale/hardening, Biosphere requirements, tiered storage, content zones, FISMA moderate capabilities, single-replica deployments, et
+
+## Roles
+
+### Project Lead [Brian O’Connor](mailto:brocono@ucsc.edu) 
+
+### Product Owner [Kevin Osborn](mailto:kosborn2@ucsc.edu) 
+
+### Technical Lead [Hannes Schmidt](mailto:hannes@ucsc.edu
```
Comment (brianraymor) [inline charters/DataStore/DataStore.md:46]: Missing closing parenthesis `[Hannes Schmidt](mailto:hannes@ucsc.edu`**)**

Comment (mweiden) [reply charters/DataStore/DataStore.md:46]: @brianraymor also: does it make sense to have emails in a heading element? Would it be clearer if the template put roles in a list?

File: charters/DataStore/DataStore.md [hunk 3]
```diff
@@ -0,0 +1,58 @@
...
+
+## In-scope
+* DSS data read and write API (PUT bundle, PUT file, GET bundle, GET file)
+* DSS data model and lifecycle (versioned bundles, etc)
+* Data lifecycle webhooks (new bundle, new file, delete bundle, delete file)
+* Checkout service - Provide continueing support for the ability to checkout the data to a local filesyste, or a personal cloud environment
+* Collections service - Maintenance and extentison of tthe ability to form arbitrary collections of objects in the Data Store
+* Multi-cloud replication of objects - This will entail extending the cloud support to more vendors (such as Microsoft Azure) as well as supporting multiple replicas within a single cloud.
+* Support for plugable indexes - Provide a standard interface for indexing modules. The plug-in interface will interact with the eventing subsystem so that all indexing systems will be notified of important data store events.This will allow a variety of 
+* Data access authorization (auth{n,z} for limited access data)
```
Comment (mweiden) [inline charters/DataStore/DataStore.md:19]: Spelling out `authentication` and `authorization` might make this more accessible to the community

File: charters/DataStore/DataStore.md [hunk 4]
```diff
@@ -0,0 +1,58 @@
...
+* HCA DCP CLI tool
+* Implementation performance scaling
+* Generic events
```
Comment (mweiden) [inline charters/DataStore/DataStore.md:22]: I would rephrase `Generic events` to make it more accessible to the community.

File: charters/DataStore/DataStore.md [hunk 5]
```diff
@@ -0,0 +1,58 @@
...
+* API Documentation for above APIs
+* Portable, cloud-neutral APIs; portable implementation of DSS on multiple clouds.
+* DevSecOps features for eventual FISMA moderate deployments (authZ/N, logging, auditing, etc).
```
Comment (mweiden) [inline charters/DataStore/DataStore.md:25]: `authentication` and `authorization`

**Review by mweiden — COMMENTED** [review]
Adding some early feedback.

**Comment by kozbo** [comment]
Thanks to both @mweiden and @brianraymor for your comments. Closing this PR in favor of a new one on my own repo for now here https://github.com/kozbo/dcp-community/pull/1
